### DISCUSSION TITLE
User input dialog

### DISCUSSION THREAD

**Discussion by Aya-Badawy89** [gama-platform/gama#108] [Q&A] [answered]
Hello Gama team,

I noticed that in Gama 1.8.2, the operator (user_input) has been changed into (user_input_dialog). Is it only a change in wording or you have added new features?

**Comment by ptaillandier** [comment, marked as answer]
Hi Aya,

I think this is the only change for the moment (the name of the operator).

Cheers,

Patrick

Le dim. 5 sept. 2021 à 03:48, Aya-Badawy89 ***@***.***> a
écrit :

**Reply by Aya-Badawy89** [reply]
I want to suggest something to add in the user input and can also be in defining parameters in general.
When inserting parameters, sometimes, parameters have a fixed total e.g., parameters representing uses on a certain piece of land, or weighting of different choices or goals which add up to 1.
In this case, it will be better NOT to insert each parameter individually because the user will have to check the total in mind. Is it possible to have the total represented in a pie and the user split it as desired. 
In the editor, the parameters can be defined by setting the value of the total, and the list of variables that make up this total. Then, in the interface, it will be displayed as a pie which, in the initial state, is equally distributed among the variables. The user can then change the distribution as required.
